ts/browser.ts

```diff
@@ -108,41 +114,50 @@ class Browser {
       protocol: "web-greeter:",
     });
     //console.log({ theme_url, url: new URL(theme_url) });
-    this.win.loadURL(`${theme_url}`);
-    this.win.setBackgroundColor("#000000");
-
-    this.win.webContents.on("before-input-event", (_event, input) => {
-      let value = nody_greeter.config.features.backlight.value;
-      if (input.type == "keyUp") return;
-      if (input.code == "BrightnessDown") {
-        Brightness.dec_brightness(value);
-      } else if (input.code == "BrightnessUp") {
-        Brightness.inc_brightness(value);
-      }
-    });
+    for (const w of this.windows) {
+      w.window.loadURL(`${theme_url}`);
```

Comment:
Here, we could check if the window is in the main monitor to load `theme_url`, either load `secondary_theme_url`.

Reply:
Interesting, we can definitely do this. How should we decide on what that secondary URL should be? I see a few options...

1. Only load a single URL (current state), and rely on theme checking metadata in javascript to decide what to display (what's currently done in 16f4bc2 for the existing themes)
2. Check for a `secondary.html` file, and load that if it exists, otherwise load `index.html`
3. Introduce a configuration file for themes as you suggest here: https://github.com/JezerM/nody-greeter/issues/10#issuecomment-1012668094

I don't have a preference either way, so am happy to implement what you suggest. If we go with option (3), i'd like a bit more guidance on how you'd want the config file to work, JSON/YAML, what filenames etc... any other configuration options...?

Reply:
I think I'll go with the third option, but others are also possible. I'm gonna push my changes later, but it would be in a YAML file, maybe `index.theme` (not sure yet), which could include the following options:

```yaml
# HTML file to use in main monitor
primary_html: "index.html"
# HTML file to use in non-main (secondary) monitors
# If not set or file does not exist, `primary_html` will be used
secondary_html: "secondary.html"
```